source/dub/dub.d

```diff
@@ -546,10 +546,11 @@ class Dub {
 			BuildSettingsTemplate tcinfo = m_project.rootPackage.recipe.getConfiguration(config).buildSettings;
 			tcinfo.targetType = TargetType.executable;
 			tcinfo.targetName = test_config;
-			// HACK for vibe.d's legacy main() behavior:
-			tcinfo.versions[""] ~= "VibeCustomMain";
-			m_project.rootPackage.recipe.buildSettings.versions[""] = m_project.rootPackage.recipe.buildSettings.versions.get("", null).remove!(v => v == "VibeDefaultMain");
-			// TODO: remove this ^ once vibe.d has removed the default main implementation
```

Comment:
This block still needs to stay for a while, since although `VibeDefaultMain` isn't endorsed since a few releases anymore, there is probably still a lot of code out there that relies on it.

Reply:
Specifically a library with a dependency on an ancient vibe version. Done.